### PR TITLE
First pass at "getting started with spk" docs

### PR DESCRIPTION
**Pull request by mtarng**
Will need to follow up with some TBD sections that are labeled, but I wanted to get this into master as a starter reference.

Part of https://github.com/microsoft/bedrock/issues/747

### CODE REVIEW

**Comment by mtarng** [comment]
Please share opinions on formatting and the structure of this guide!

File: docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+  touch README.md
+  echo "This is the Flux Manifest Repository." >> README.md
+  git add -A
+  git commit -m "Initializing Materialized Manifests repository with a README."
+  git push -u origin --all
+  ```
+
+#### High Level Definition Repository
+
+This repository holds all the bedrock "High Level Defenition" (HLD) yaml files
```
Comment (dennisseah) [inline docs/getting-started.md]: "Defenition" typo

File: docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+and associated configurations. These HLDs and configs are consumed via
+[fabrikate](https://github.com/microsoft/fabrikate) to produce kubernetes
+manifests. This is typically done via an Azure DevOps pipeline, and the
+manifests output by fabrikate are placed into the Materialied Manifests
```
Comment (dennisseah) [inline docs/getting-started.md]: "Materialied" typo

**Review by dennisseah — CHANGES_REQUESTED** [review]

File: docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+3. [Initialize spk](#initializing-spk)
+
+#### Generating Personal Access Token
+
+Generate a new Personal Access Token (PAT) to grant `spk` permissions in the
+Azure Devops Project. Please grant PAT the following permissions:
+
+- Build (Read & execute)
+- Code (Read, write, & manage)
+- Variable Groups (Read, create, & manage)
```
Comment (dennisseah) [inline docs/getting-started.md:111]: I stop using full access and use these permissions. And it works ok. thanks

**Review by dennisseah — APPROVED** [review]

**Review by evanlouie — APPROVED** [review]
looks good so far 👍

**Pull request merged**